build.ps1

```diff
@@ -0,0 +1,6 @@
+del -Recurse -Force .\build\*
+gofmt -e -s -w .
+go mod tidy
+$LDFLAGS="-ldflags=-X 'main.buildTime=$(date)' -X 'main.lastCommit=$(git rev-parse HEAD)' -X 'main.semanticVersion=$(git describe --tags --dirty=-dev)'"
+go build -o build/ ${LDFLAGS} ./cmd/celestia
+go build -o build/ ./cmd/cel-key
```

Comment:
It will be appreciated if you have any suggestions to improve it. Because the 'make' on Win is really old, it hasn't been updated for many years, I don't want to use it, that's why I created this script. But I don't think it is worth spending a lot of time to create a pretty script since it's for s specific platform, that's why it looks like that.